authentication/authentication-common/src/test/java/io/mosip/authentication/common/service/impl/IdServiceImplTest.java

```diff
@@ -298,6 +297,10 @@ public void updateVidStatusTest() throws IdAuthenticationBusinessException {
 		Mockito.when(identityRepo.existsById(vid)).thenReturn(true);
 		Mockito.when(identityRepo.getOne(vid)).thenReturn(entity);
 		ReflectionTestUtils.invokeMethod(idServiceImpl, "updateVIDstatus", vid);
+
+		entity.setTransactionLimit(0);
+		Mockito.when(identityRepo.getOne(vid)).thenReturn(entity);
+		ReflectionTestUtils.invokeMethod(idServiceImpl, "updateVIDstatus", vid);
 	}
 
```

Comment:
Please add test cases for below:
* transacstion limit is 1
* transaction limit is 2
* transaction limit is -1
* transaction limit is null

Reply:
I've added all the above cases.